### ISSUE TITLE
read_tsv: skip rows while reading based on a condition?

### ISSUE THREAD

**Issue by peranti** [tidyverse/readr#1157]
Hi, Thanks for this wonderful package. 

May I ask if there is a possibility, within the functionality of this package or tidyverse family, on skipping some rows based on a condition while reading the file using a function like `read_tsv`?

For example, skip rows if a column contains value like `"."`, `"NA"`, etc? 

This kind of functionality will be very useful while reading files with more than 10M rows.

**Comment by jimhester** [comment]
No, it is not possible nor likely to become so. I would suggest you pre-filter the file using command line tools, e.g. `readr::read_tsv(pipe("grep -bv "NA"))`
